src/hosting/api.ts

```diff
@@ -160,7 +160,7 @@ export interface Version {
  */
 export function normalizeName(s: string): string {
   // Using a regex replaces *all* bad characters.
-  return s.replace(/[/:_]/g, "-");
+  return s.replace(/[/:_#]/g, "-");
```

Comment:
Instead of "*all* bad characters" maybe specify which characters are being replaced

Reply:
The jsdoc comment is now updated to include `#`. But this comment is about using regex to replace the characters rather than another mechanism. I made it more specific.